k8s.io/test.py

```diff
@@ -260,7 +260,7 @@ def test_docs(self):
 
     def test_examples(self):
         for base in ('examples.k8s.io', 'examples.kubernetes.io'):
-            self.assert_temp_redirect(base, 'https://github.com/kubernetes/kubernetes/tree/master/examples/')
+            self.assert_temp_redirect(base, 'https://github.com/kubernetes/examples/tree/master')
```

Comment:
test fails:
```
======================================================================
FAIL: test_examples (__main__.RedirTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test.py", line 276, in test_examples
    self.assert_temp_redirect(base, 'https://github.com/kubernetes/examples/tree/master')
  File "test.py", line 80, in assert_temp_redirect
    self.assert_multischeme_redirect(partial_url, expected_loc, 302, **kwargs)
  File "test.py", line 77, in assert_multischeme_redirect
    scheme + '://' + partial_url, expected_loc, expected_code, **kwargs)
  File "test.py", line 72, in assert_scheme_redirect
    self.assertEqual(resp.getheader('location'), expected_loc)
AssertionError: 'https://github.com/kubernetes/examples/tree/master/' != 'https://github.com/kubernetes/examples/tree/master'
```

Reply:
for thoroughness it might also be worth expanding this to include a case that verifies the request URI on examples.k8s.io/ is handled, too.

Reply:
Fixed and added assert for tailing slash